src/app.scss

```diff
@@ -4,5 +4,8 @@
 @import 'carbon-components/scss/globals/scss/vars.scss';
 
 @import './components/TutorialHeader/tutorial-header.scss';
+@import './components/info/info.scss';
+
```

Comment:
Should be `@import './components/Info/info.scss';`

Reply:
changed it.